### DISCUSSION TITLE
How do I make sure I capture new comments on previously-downloaded threads + how can I export the JSON?

### DISCUSSION THREAD

**Discussion by Geethebluesky** [MalloyDelacroix/DownloaderForReddit#352] [Q&A] [answered]
I'm tentatively fiddling with settings but am not sure I'm downloading the most recent comments on older posts I might have downloaded a while ago. What settings should I be using for that please?

And is there a way to mass-export the reddit-format JSON of posts + comments I've downloaded? I'm hoping for the same format as what comes up when you add .json to a link, for example:

https://old.reddit.com/r/Catswhoyell/comments/14i11f2/caught_my_cat_stormy_mid_yell.json

Thanks.

**Comment by MalloyDelacroix** [comment, marked as answer]
To download new comments on older posts you've already downloaded, uncheck the "Limit by date" checkbox under "Post Download Settings" for any user/subreddit you want to download the comments of.

You can export almost all data to a JSON file, but the format will be different than in the example you provided.  You can export a user/subreddit and all of the related posts and comments by right-clicking the user/subreddit in the respective list, then clicking "Export <user/subreddit name>".  Follow the steps in the wizard to export what you need.  If you only want to export posts, you can select the posts you want to export from the Database View dialog and export them by right-clicking and selecting the export option.

**Reply by Geethebluesky** [reply]
Thanks so much. And thanks for making this tool :)
